charts/athens-proxy/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: athens-proxy
-version: 0.3.5
+version: 0.3.6
 appVersion: 0.6.1
```

```suggestion
appVersion: 0.7.1
```

If going ahead with a new patch release, is this the PR to bump this?

---

Also needs a bump at https://github.com/gomods/athens/pull/1428/files#diff-71a5c45e593365379cc2572e15c7db38L5